packages/vscode-extension/src/extension.ts

```diff
@@ -358,9 +358,7 @@ export async function activate(context: vscode.ExtensionContext) {
   const userDataSelector = {
     language: "plaintext",
     scheme: "file",
-    pattern: isMultiEnvEnabled()
-      ? `**/.${ConfigFolderName}/${StatesFolderName}/*.userdata`
-      : `**/.${ConfigFolderName}/*.userdata`,
+    pattern: "**/*.userdata",
```

Comment:
Does this mean we allow users to edit current project user data from backup files?

Reply:
To be more accurate, to let user read/find out previous config in .userdata in .backup folder.